controllers/apply/expiry.js

```diff
@@ -201,14 +201,17 @@ async function deleteExpiredApplications(expiredApplications) {
 
             const dbStatus = await PendingApplication.deleteBatch(ids);
 
-            expiredApplications.forEach(application => {
+            const status = expiredApplications.map(application => {
                 logger.info(`Deleting expired application`, {
                     formId: application.formId,
                     applicationStatus: application.currentProgressState
                 });
+                return {
+                    applicationDeleted: true
+                };
             });
 
-            return dbStatus === expiredApplications.length;
+            return dbStatus === expiredApplications.length ? status : false;
```

Comment:
Not sure I follow this line. If we have expired applications then when would `dbStatus === status` be truthy? Is this not comparing two different types of value?

Reply:
It's a bit confusing but I believe `dbStatus` returns the number of rows affected by the operation (in this case the bulk deletion above), so it always returns `true` unless something weird happened (eg. it deleted some but not all of the applications). We do something similar in the `sendExpiryEmails` function below this to confirm it did as expected.